cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -146,11 +146,21 @@
         left: 50%;
         top: 50%;
         transform: translate( -50%, -50% );
+        
+        &:focus-visible {
+            outline: 1px dotted @pacific;
+            outline-offset: 1px;
+        }
     }
 
     &__bottom-right {
         right: unit( @grid_gutter-width / 2 / @play-btn-icon-size, em );
         bottom: unit( @grid_gutter-width / 2 / @play-btn-icon-size, em );
+
+        &:focus-visible {
```

Comment:
To ensure the focus state is kept in-sync for both button positions, it's probably best to define this in one place. To do that you'd make a block below:

```
 &__center,
 &__bottom-right {
   &:focus-visible {
     outline: 1px dotted @pacific;
     outline-offset: 1px;
   }
 }
```